src/config/i18n/en-us.json

```diff
@@ -356,5 +356,6 @@
     "finalizationPoint": "Finalization Point",
     "finalizedHash": "Finalized Hash",
     "nodeDetailError": "Failed to fetch node detail.",
-    "extendGraphicValue": "Info"
+    "extendGraphicValue": "Info",
+    "votingNode": "(Voting)"
```

Comment:
I think we should add more detail. Somithing like this:

`Voting node. Finalized height may be behind` (or ahead)

Reply:
And `( )` I think should be hardcoded in the component